resources/views/character/partials/header.blade.php

```diff
@@ -1,4 +1,14 @@
 <ul class="no-bullet no-indent">
+    @if (isset($character->pivot->created_at))
+    <li>
+        <ul class="list-inline">
+            <li class="list-inline-item text-muted small">
+                  Recieved: <span class="js-watchable-timestamp js-timestamp-title" data-timestamp="{{ $character->pivot->created_at }}"></span> ago
+            </li>
+        </ul>
+    </li>
+    @endif
+    <li/>

```

Comment:
I think this should be removed from the PR. This template is for displaying character names on pages generated by PHP, and I can't see a use case where we'd want this to appear above the character's name.

Reply:
I do not think this feature should be removed. This provides the time the item was received on the Loot Item page as shown in the screen shot below. I am happy to change how its implemented if you have suggestions. But this feature needs to exist on the item page. 


Example: 
![image](https://user-images.githubusercontent.com/69526309/90270466-18ca2c80-de0f-11ea-9132-0e3c8508b05d.png)

Reply:
Okay, I see where it's useful now. In that case, just remove the trailing `<li/>` please.

Reply:
Removed the <li/>. That was as artifact of a different implementation.